meinberlin/apps/votes/api.py

```diff
@@ -81,18 +87,13 @@ def check_permissions(self, request):
         if not hasattr(self, 'token'):
             self.permission_denied(
                 request,
-                message=_('No token given.')
+                message=_('No token given or token not valid for module.')
             )
         elif not self.token.is_active:
             self.permission_denied(
                 request,
                 message=_('Token is inactive.')
             )
-        elif not self.token.module == self.module:
-            self.permission_denied(
-                request,
-                message=_('Token not valid for module.')
-            )
```

Comment:
Since in dispatch the token will only be derived from the db if the module matches the token, this case cannot happen anymore